data/loftwah.json

```diff
@@ -1,31 +1,30 @@
 {
   "name": "Dean Lofts",
   "type": "personal",
+  "displayStatsPublic": true,
   "bio": "DevOps Engineer, Music Producer, and big fan of Open Source.",
   "avatar": "https://github.com/loftwah.png",
+  "tags": [
+    "AWS",
+    "Beats",
+    "DevOps",
+    "Music",
+    "Open Source",
+    "Ruby",
+  ],
+    "socials": [
+    { "icon": "FaGithub", "url": "https://github.com/loftwah" },
+    { "icon": "FaTwitter", "url": "https://twitter.com/loftwah" },
+    { "icon": "FaYoutube", "url": "https://youtube.com/loftwahthebeatsmiff" }
+  ],
   "links": [
     {
-      "name": "Github",
-      "url": "https://github.com/loftwah",
+      "name": "Linux for Pirates!",
+      "url": "https://github.com/loftwah/linux-for-pirates",
       "icon": "FaGithub"
     },
     {
-      "name": "Twitter",
-      "url": "https://twitter.com/loftwah",
-      "icon": "FaTwitter"
-    },
-    {
-      "name": "Instagram",
-      "url": "https://instagram.com/loftwah",
-      "icon": "FaInstagram"
-    },
-    {
-      "name": "Facebook",
-      "url": "https://facebook.com/loftwah",
-      "icon": "FaFacebook"
-    },
-    {
-      "name": "Songs I've produced ",
+      "name": "Songs I've produced",
       "url": "https://www.youtube.com/watch?v=txk5S_r7vkg&list=PLKBAUoCO_FtkHiwRzyGzfhauIhNMBFw66",
       "icon": "FaYoutube"
     },
```

Comment:
Shouldn't line 44 (Spotify icon) be updated to `FaSpotify` as well?

Reply:
I noticed that, but I could see a `FaStpofy` that could be used. I am happy to change this, but it was already `spotify` previously when the others were updated automatically. I'm not sure if Spotify is supported by it.

Reply:
Fa Spotify icons are available, you can search on the icons page https://linkfree.eddiehub.io/icons

<img width="656" alt="Screenshot 2023-01-02 at 18 49 58" src="https://user-images.githubusercontent.com/624760/210269448-de8a9038-b8eb-40da-8eb1-04766a1f5556.png">

Reply:
Nice I'll fix this up today.